### DISCUSSION TITLE
Help with rigtools? V127

### DISCUSSION THREAD

**Discussion by Balistic123** [3kh0/ext-remover#1225] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I’m on v127 and when I use rigtools and try to execute something some code works but for example when I try to execute prompt(“hi”) it executes on the rigtools page but doesn’t on any other page? I’m using lockdown browser extension

**Comment by schoolexploitkid** [comment, marked as answer]
try using the lockdown browser to execute ltbeef inside rigtools to disable all extensions. i currently have v129, so i can't show you.

**Reply by Balistic123** [reply]
> try using the lockdown browser to execute ltbeef inside rigtools to disable all extensions. i currently have v129, so i can't show you.

Thats not my issue, my issue is nothing really executes on other pages

**Comment by Balistic123** [comment]
Thats not my issue, my issue is nothing really executes on other pages

**Comment by Charlieis1nsan3** [comment]
my friend helped make this his name entrpix err her :| he sh idk is a trans
